recipes/pyppmd/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pyppmd" %}
+{% set version = "0.15.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3771a65699caee0f5b97cedada74a52b9ba4061bd8ad150d0115defb291c4581
+
+build:
+  number: 0
+  script:
+    # The tarball shouldn't have egg-info--this one has paths in SOURCES.txt
+    # that break the build on windows.
+    - rm -rf src/pyppmd.egg-info
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+    - setuptools >=45.0
+    - setuptools-scm >=6.0.1
+    - importlib_metadata
```

Comment:
`importlib_metadata` doesn't look like it should be needed at build-time.

Reply:
Done!